priv/repo/migrations/20160322222933_add_null_chat_provider.exs

```diff
@@ -0,0 +1,15 @@
+defmodule Cog.Repo.Migrations.Addnullchatprovider do
```

Comment:
Sorry to nitpick, but do you mind camelizing this module name as well? Once that's done :+1: to merge.